src/main/java/net/nikr/eve/jeveasset/gui/shared/filter/FilterGui.java

```diff
@@ -479,16 +495,27 @@ protected String getFilterName() {
 		return filterSave.show(new ArrayList<>(filterControl.getFilters().keySet()), new ArrayList<>(filterControl.getDefaultFilters().keySet()));
 	}
 
+	/**
+	 * Loop though set update listeners and trigger their action.
+	 */
+	private void fireSettingsUpdate() {
+		for (SettingsUpdateListener listener : filterControl.getSettingsUpdateListenerList()) {

```

Comment:
As everything else it's a minor issue, but, it pops up in my IDE as a warning.
Would just suggest rename the local variable it to something like `settingsListener` to avoid confusion.

Reply:
Yeah my guess is the IDE thinks we want to use that method to expose the object. settingsListener or settingsUpdateListener?

Reply:
I think it warns of duplicated names because, using the same name in both the classes and sub classes are just bad, I guess, it may lead to confusion when reading the code.
both `settingsListener ` and `settingsUpdateListener` are fine, take your pick.